.gitignore

```diff
@@ -27,3 +27,5 @@ integration-test/artifacts/
 # One offs
 test/App/Fossa/VSI/DynLinked/testdata/hello_standard
 test/App/Fossa/VSI/DynLinked/testdata/hello_setuid
+test/App/Fossa/VSI/DynLinked/testdata/hello_standard.exe
+test/App/Fossa/VSI/DynLinked/testdata/hello_setuid.exe
```

Comment:
[nit]: I'd prefer to combine these with the previous entries using `*`.

Reply:
fixed in https://github.com/fossas/fossa-cli/pull/857/commits/adb83370d7b1cd88db3c89dc87c7806d7fe30022